src/appforms/src/core/020-utils02webcam.js

```diff
@@ -24,10 +24,18 @@ appForm.utils = function (module) {
     checkEnv();
     _html5Camera(params, cb);
   }
+
   function cancelHtml5Camera() {
-    if (localMediaStream) {
-      localMediaStream.stop();
-      localMediaStream = null;
+    if (localMediaStream){
+      if (localMediaStream.stop) {
+        localMediaStream.stop();
+        localMediaStream = null;
+      } else{
+        var tracks = localMediaStream.getTracks();
```

Comment:
Ok, anyway, IMO, I think it's safer to just do `tracks && tracks.length`. God knows if other browsers will follow the draft...